README.md

```diff
@@ -26,7 +26,22 @@
           "app_category_type": "public.app-category.productivity"
         },
         "linux": {
-          "icon": "resources/linux/<your-project-id>.png"
+          "icon": "resources/linux/<your-project-id>.png",
+          "debian_config": {
+            "section": "Databases",
+            "depends": [
+              "python"
+            ],
+            "suggests": [
+              "libgnome-keyring0"
+            ]
+          },
+          "redhat_config": {
+            "category": "Applications/Databases",
+            "requires": [
+              "libXScrnSaver(x86-64)"
```

Comment:
Aha! Sorry for delay, didn't get any notifications for your comments above :(

For `libXScrnSaver`: excellent!  Thanks for all of the detail here.  I suspect this is being caused by missing deps on Evergreen.  Will follow-up with a BUILD ticket.